kite-data/kite-data-crunch/src/test/java/org/kitesdk/data/crunch/TestCrunchDatasetsHBase.java

```diff
@@ -163,4 +164,5 @@ private void checkRecords(Dataset<GenericRecord> dataset, int count, int start)
     }
   }
 
+*/
 }
```

Comment:
Yeah, I went back and forth on that. I was thinking I might try and have another go at getting HBase to work, but I can just restore the file if that's the case. I'll remove them.